### PR TITLE
49191 URL corrections & page titles

### PR DESCRIPTION
**Pull request by mohamme15315**
Added code for hash link and URL route / title

### CODE REVIEW

File: UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/Shared/_MSILayout.cshtml [hunk 1]
```diff
@@ -24,6 +24,9 @@
     </div>
 </div>
 
+
+@*<script type="text/javascript" src="@Url.Content("~/js/Custom/Common.js")"></script>*@
+
```
Comment (usha13529) [inline UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/Shared/_MSILayout.cshtml:29]: Please remove this commented code.

File: UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/Shared/_NMHorizontalTab.cshtml [hunk 2]
```diff
@@ -6,9 +6,11 @@
         <a class="nav-link @(ViewContext.RouteData.Values["Action"].ToString() == "ShowDailyFiles" ? "active":"")" tabindex="@(ViewContext.RouteData.Values["Action"].ToString() == "ShowDailyFiles" ? "0":"-1")" id="daily-tab" asp-action="ShowDailyFiles" asp-controller="NoticestoMariners" role="tab">Daily</a>
     </li>
     <li class="nav-item" role="presentation">
-        <a href="" class="nav-link" id="cumulative-tab" data-bs-toggle="tab" data-bs-target="#cumulative" role="tab" aria-controls="cumulative" aria-selected="false" tabindex="-1">Cumulative</a>
+        @*<a href="#cumulative" class="nav-link" id="cumulative-tab" data-bs-toggle="tab" data-bs-target="#cumulative" role="tab" aria-controls="cumulative" aria-selected="false" tabindex="-1">Cumulative</a>*@
```
Comment (usha13529) [inline UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/Shared/_NMHorizontalTab.cshtml:9]: Please remove this commented code.

File: UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/Shared/_NMHorizontalTab.cshtml [hunk 3]
```diff
@@ -6,9 +6,11 @@
...
+        <a class="nav-link @(ViewContext.RouteData.Values["Action"].ToString() == "Cumulative" ? "active":"")" tabindex="@(ViewContext.RouteData.Values["Action"].ToString() == "Cumulative" ? "0":"-1")" id="cumulative-tab" asp-action="Cumulative" asp-controller="NoticestoMariners" role="tab">Cumulative</a>
     </li>
     <li class="nav-item" role="presentation">
-        <a href="" class="nav-link" id="annual-tab" data-bs-toggle="tab" data-bs-target="#annual" role="tab" aria-controls="annual" aria-selected="false" tabindex="-1">Annual</a>
+        @*<a href="#annual" class="nav-link" id="annual-tab" data-bs-toggle="tab" data-bs-target="#annual" role="tab" aria-controls="annual" aria-selected="false" tabindex="-1">Annual</a>*@
```
Comment (usha13529) [inline UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/Shared/_NMHorizontalTab.cshtml:13]: Please remove this commented code.

File: UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformationAdmin.Web/appsettings.Development.json [hunk 4]
```diff
@@ -7,8 +7,15 @@
...
         "ConnectionString": ""
+
     },
     "RadioNavigationalWarningConfiguration": {
         "AdminListRecordPerPage": 20
+    },
+    "AzureAdB2C": {
+        "Instance": "",
+        "ClientId": "",
+        "Domain": ""
```
Comment (usha13529) [inline UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformationAdmin.Web/appsettings.Development.json:18]: Please remove AzureAdB2C section form admin app config as its not relevant to changes done for this PBI and also admin app requires AzureAd and not AzureAdB2C settings.

File: Tests/AutoTests/FunctionalTest/functional-weeklyfiledownload.spec.ts [hunk 5]
```diff
@@ -27,4 +30,18 @@ test.describe("Goto maritime-safety-information Notice To Mariner Page to Check
     await noticeFileDownload.goToDailyFile();
     await noticeFileDownload.checkDailyFileDownload();
   })
+  test('Does the Notice to mariner Page url are displayed with page title',async ({page}) => {
```
Comment (usha13529) [inline Tests/AutoTests/FunctionalTest/functional-weeklyfiledownload.spec.ts:33]:
```suggestion
  test('Does the Notices to Mariners Page url's are displayed with page title', async ({page}) => {
```

**Review by usha13529 — CHANGES_REQUESTED** [review]
Few minor comments to fix please.

File: Tests/AutoTests/FunctionalTest/functional-weeklyfiledownload.spec.ts (hunk 5 above)
Comment (jigar15426) [reply Tests/AutoTests/FunctionalTest/functional-weeklyfiledownload.spec.ts]: resolved

File: UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/Shared/_NMHorizontalTab.cshtml (hunk 2 above)
Comment (mohamme15315) [reply UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/Shared/_NMHorizontalTab.cshtml]: Removed commented lines

File: UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/Shared/_NMHorizontalTab.cshtml (hunk 3 above)
Comment (mohamme15315) [reply UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/Shared/_NMHorizontalTab.cshtml]: Removed commented lines

File: UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformationAdmin.Web/appsettings.Development.json (hunk 4 above)
Comment (mohamme15315) [reply UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformationAdmin.Web/appsettings.Development.json]: removed AzureAdB2C section

**Review by usha13529 — APPROVED** [review]
Thanks for the changes @mohamme15315 & @jigar15426. Changes looks good to me.

**Review by barrie-cooper — APPROVED** [review]
LGTM, thanks @mohamme15315

Will merge once we sort the dependency check issues out.

**Pull request merged**